api/v1alpha1/hybrisapp_types.go

```diff
@@ -41,6 +41,15 @@ type HybrisAppSpec struct {
 
 	// Hybris app repository source location
 	SourceRepoContext string `json:"sourceRepoContext,omitempty"`
+
+	// Hybris app repository local.properties override location
+	SourceRepoLocalPorpertiesOverride string `json:"sourceRepoLocalPropertiesOverride,omitempty"`
+
+	// Hybris app Apache server.xml jvmRoute name
+	ApachejvmRouteName string `json:"apachejvmRouteName,omitempty"`
+
+	// Hybris app ANT tasks
+	HybrisANTTaskNames string `json:"hybrisANTTaskNames,omitempty"`
```

Comment:
```suggestion
	HybrisANTTaskNames *string `json:"hybrisANTTaskNames,omitempty"`
```

Reply:
I do not see these types being different from the existing ones, that the existing design needs to be altered. There is a check happening in assemble script, and the status gets ignored if the values are empty.